packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -141,13 +164,24 @@ const UserForm: FC<UserFormProps> = (props: UserFormProps) => {
           </Form.Item>
 
           <Form.Item id="enabled" name="enabled" label={lang.ENABLE_USER}>
-            <Radio.Group options={status} name="enabled"></Radio.Group>
+            <Radio.Group
+              options={status}
+              name="enabled"
```

Comment:
nitpick: do not need name here since it goes on Form.Item

Reply:
I'm using the selectors in tests